core/templates/dev/head/pages/exploration_editor/simple_editor_tab/StatesToQuestionsService.js

```diff
@@ -24,11 +24,16 @@ oppia.factory('StatesToQuestionsService', [
     var SUPPORTED_INTERACTION_TYPES = [{
       id: 'MultipleChoiceInput',
       name: 'Multiple choice'
+    },
+    {
+      id: 'ItemSelectionInput',
+      name: 'Item Selection Input'
     }];
     var INTERACTION_ID_END_EXPLORATION = 'EndExploration';
 
     var allowedInteractionIds = SUPPORTED_INTERACTION_TYPES.map(
       function(interactionTypeData) {
+
```

Comment:
There are a lot of randomly-added lines like this scattered all over this PR. Please look at the diff on GitHub and remove what's not needed.